carla-carma-integration/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+FROM ubuntu:16.04
```

Comment:
I think this is redundant, each "FROM" overwrites the prior one. To my recollection, ros:kinetic is already based on ubuntu:16.04 anyway.

Reply:
The redundant line has been removed